site/components/form/DateSelector.tsx

```diff
@@ -33,7 +33,7 @@ const inputBox = <T extends object>(
     disabled: boolean,
 ) => (
     <div className="govuk-date-input flex flex-row [&_.MuiSvgIcon-root]:fill-govBlue">
-        <div className="govuk-date-input__item govuk-!-margin-right-0">
+        <div className="govuk-date-input__item govuk-!-margin-right-0]">
```

Comment:
Does the "]" do anything here?

Reply:
Alignment looks very slightly off for the date boxes as they are all different lengths. Perhaps we could add a min width to this input, something like: ```<div className="govuk-date-input__item govuk-!-margin-right-0 min-w-[240px]">```

This would mean increasing the min width for for the divs containing the DateSelector components in [disruptionId].page.tsx - see other comment

I think we should also remove the "]" as it adjusts the alignment in a weird way.

Reply:
done